src/queues.c

```diff
@@ -213,6 +213,37 @@ void history_push(notification *n)
                 g_queue_push_tail(history, n);
 }
 
+gint64 queues_get_next_datachange(gint64 time)
```

Comment:
Yes, there seems to be no rationale for this. I don't know why I thought this would be neccessary. Maybe because I have to pass the output of `x_is_idle`.